cmd/kepval/main.go

```diff
@@ -24,6 +24,8 @@ import (
 	"k8s.io/enhancements/pkg/kepval/keps"
 )
 
+// TODO: Can we remove this file?
```

Comment:
@johnbelamaric - what do you think about this one?
Do you know if people are using this binary?

I think that changing this binary to what the test itself is doing might not be the best option (because e.g. we're hardcoing prod-readiness directory, which makes sense for test, but not necessary for the binary).
OTOH, without that, it's not a full solution.

So while things like "kepctl" are really binaries, here what we really need is a test.

WDYT?

Reply:
yeah, I don't know what this is for